apps/passport-server/test/genericIssuance.spec.ts

```diff
@@ -107,56 +111,67 @@ describe("Generic Issuance", function () {
 
   const lemonadeBackend = new LemonadeDataMocker();
 
-  /**
-   * TODO: test ingestion of the data we'll need for production.
-   */
-  const EdgeCityDenver = lemonadeBackend.addEvent("Edge City Denver");
+  const EdgeCityLemonadeAccount = lemonadeBackend.addAccount(
+    "edge-city-client-id"
+  );
+
+  const EdgeCityDenver = EdgeCityLemonadeAccount.addEvent("Edge City Denver");
 
   /**
-   * Attendee ticket tier. In reality there will be several.
+   * Attendee ticket type. In reality there will be several.
    *
-   * TODO: test that we can handle several attendee tiers.
+   * TODO: test that we can handle several attendee types.
```

Comment:
i think this TODO comment is now outdated, since you have an attendee and a bouncer ticket type in this test